GlimpseImputationPipeline/README.md

```diff
@@ -0,0 +1,122 @@
+- **File input_vcf**: VCF to be imputed, single sample or multi sample. Sites not present in the reference panel will be ignored, as are sites present in the reference panel and not in the input VCF. You may only set either this argument or `crams`.
+- **File input_vcf_index**: Index to `input_vcf`.
+- **Array[File]? crams**: Array of CRAM files as an alternative to `input_vcf`.
+- **Array[File]? cram_indices**: Indices to `crams`. You may only set either this argument or `input_vcf`.
+- **Array[String] sample_ids**: Sample IDs to be assigned when using CRAM input.
+- **File? fasta**: *Highly recommended when using CRAM input, otherwise the task will attempt to download the reference from a remote server, which can be incredibly slow.
+- **File? fasta_index**: Index to `fasta`.
+- **String output_basename**: Basename for imputed output VCF.
+- **File ref_dict**: By default, GLIMPSE2 only adds the contig names defined by the `contig_regions` input in GlimpseSplitReference to the VCF header's sequence dictionary. In order to make the file compatible with downstream tools the sequence dictionary will be replaced with this input.
+- **Boolean impute_reference_only_variants = false**: See [GLIMPSE2 documentation](https://odelaneau.github.io/GLIMPSE/docs/documentation/phase/#input-parameters).
+- **Boolean call_indels = false**: See [GLIMPSE2 documentation](https://odelaneau.github.io/GLIMPSE/docs/documentation/phase/#bamcram-options-and-filters).
+- **Int? n_burnin**: See [GLIMPSE2 documentation](https://odelaneau.github.io/GLIMPSE/docs/documentation/phase/#model-parameters).
+- **Int? n_main**: See [GLIMPSE2 documentation](https://odelaneau.github.io/GLIMPSE/docs/documentation/phase/#model-parameters).
+- **Int? effective_population_size**: See [GLIMPSE2 documentation](https://odelaneau.github.io/GLIMPSE/docs/documentation/phase/#model-parameters).
+- **Int preemtible = 1**: Number of preemptible attempts
+- **File? monitoring_script**: Optional path to monitoring script. If ommitted, no monitoring will occur and the `split_reference_monitoring** output will not be available.
+
+### Output
+
+- **File imputed_vcf**: Single imputed VCF that covers all regions defined in the `contig_regions` input in GlimpseSplitReference. The name of the file is the basename of `input_vcf` with `.imputed.vcf.gz` added.
+- **File imputed_vcf_index**: Index to `imputed_vcf`.
+- **Array[File?] glimpse_phase_monitoring**: A monitoring log for each parallelized chunk if the `monitoring_script` input is set, otherwise null.
+- **File? glimpse_ligate_monitoring**: A monitoring log for the ligate task if the `monitoring_script` input is set, otherwise null.
+
+## Glimpse2MergeBatches
+
+This workflow merges multiple batches of imputed multi-sample VCFs into one and recalculates the AF and INFO score annotations based on [this discussion](https://github.com/broadinstitute/hydro.gen/issues/900).
```

Comment:
links to private repo from a public repo.  Could instead copy the math into this README.

Reply:
Added to appendix